### PR TITLE
Faster test run

### PR DESCRIPTION
**Pull request by unparalleled-js**
Make the tox unit tests command faster
Adds section in contributing for how to run integration tests:

```pytest -m "integration"```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,6 +124,12 @@ This will also test that the documentation build passes and run the style checks
 $ tox -e py
 ```
 
+If you want to run the integration tests in your current python environment, you can do:
+
+```bash
+pytest -m "integration"
```
Comment (unparalleled-js) [inline CONTRIBUTING.md:130]: This is the benefit of marking the tests as integration

File: tox.ini [hunk 2]
```diff
@@ -19,7 +19,7 @@ commands =
     # -l: show locals in tracebacks
     # --tb=short: short traceback print mode
     # --strict: marks not registered in configuration file raise errors
-    pytest --cov=code42cli --cov-report xml -v -rsxX -l --tb=short --strict -m "not integration"
+    pytest --cov=code42cli --cov-report xml -v -rsxX -l --tb=short --strict --ignore=integration
```
Comment (unparalleled-js) [inline tox.ini:22]: This is the benefit of _still_ keeping the integration tests in their own directory... Ignoring a whole directory is significantly faster than ignoring individual tests with a `pytest mark`.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kiran-chaudhary) [reply CONTRIBUTING.md:130]: We might want to add that to run integration tests "Set environment variables C42_USERNAME and C42_PW with CCA credentials".

File: tox.ini (hunk 2 above)
Comment (kiran-chaudhary) [reply tox.ini:22]: This is nice, as I marked the individual test class as `integration`, some of the negative tests which were kind of unit tests still exists in `integration` directory, we might want to move them back to appropriate unit test files.

I can create a pull request against this branch with the changes and then we can merge this to master to make sure all unit tests are executed.

**Review by kiran-chaudhary — COMMENTED** [review]
We need some more refactoring to get this work as desired.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kiran-chaudhary) [reply CONTRIBUTING.md:130]: I added this information in `py42` repo but somehow missed it here.

**Comment by unparalleled-js** [comment]
@kiran-chaudhary Yeah, unit tests should probably not be in the integration test folder!  Thanks, you could also just open that PR against Master.

**Review by kiran-chaudhary — APPROVED** [review]
LGTM, as discussed I will pull in another PR to move tests to their appropriate locations.

**Pull request merged**